lang/en/texts/update-a-product-from-the-mobile-app.html

```diff
@@ -0,0 +1,666 @@
+<h1>Update a product from the mobile app</h1>
+
+<div class="row">
+	<div class="small-12 medium-4 columns">
+        <img src="https://upload.wikimedia.org/wikipedia/commons/1/17/FAQ_icon_%28Noun_like%29.svg">
+        <div class="card-section">
+          <h4><a href="/faq">Frequently Asked Questions</a></h4>
+          <p>It has an easy to override visual style, and is appropriately subdued.</p>
```

Comment:
What does the text mean? It's repeated several times.

Reply:
it's placeholder text :-)